pkg/controller/nstemplateset/nstemplateset_controller.go

```diff
@@ -0,0 +1,353 @@
+package nstemplateset
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/codeready-toolchain/member-operator/pkg/template"
```

Comment:
I have run `make format-go-code` and there is no change.  Rest is personal preference.

Reply:
We discussed about it during the last sync call and agreed on this format/splitting in sections.

Reply:
It would be useful if we can automate that (ideally formatting or at least add some linter for the PRs) somehow (@tinakurian wink wink). But IMO it's not that critical to spend too much time on it.
I would vote for this import rule as "nice to have" rather than a hard requirement until we have some kind of automation.